src/test/java/seedu/address/model/person/EmailTest.java

```diff
@@ -1,61 +0,0 @@
-package seedu.address.model.person;
-
-import static org.junit.jupiter.api.Assertions.assertFalse;
-import static org.junit.jupiter.api.Assertions.assertTrue;
-import static seedu.address.testutil.Assert.assertThrows;
-
-import org.junit.jupiter.api.Test;
-
-public class EmailTest {
```

Comment:
Why are we removing this ah?

Reply:
My bad this was unintended thanks for catching it.